new-pipeline/src/dashboards.js

```diff
@@ -1048,8 +1048,14 @@ function updateOSs() {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
-  var imgUrl = "https://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png"
```

Comment:
This was the only use of this file, can you remove the actual file too?

Reply:
yeah sure